bazel/foreign_cc/luajit.patch

```diff
@@ -149,7 +149,7 @@ index 00000000..1201542c
 +    shutil.copytree(src_dir, os.path.basename(src_dir))
 +    os.chdir(os.path.basename(src_dir))
 +
-+    os.environ["MACOSX_DEPLOYMENT_TARGET"] = "10.6"
++    os.environ["MACOSX_DEPLOYMENT_TARGET"] = "10.8"
```

Comment:
this is the change

Reply:
Why is there such a large change to the rest of the Python and resulting msvcbuild.bat? CC @phlax

Reply:
Some whitespace issue. Should be fixed now